robot/basestation/app.py

```diff
@@ -95,6 +95,24 @@ def science():
     """Science page."""
     return flask.render_template("Science.html", roverIP=fetch_ros_master_ip())
 
+# routes for science page
+@app.route('/numSections')
+def numSections():
+    return '4'
+
+@app.route('/initialSection')
+def initialSection():
+    return '0'
+
+@app.route('/rotatePos')
+def rotatePos():
+    time.sleep(1) # wait 1 second
+    return 'done'
+
+@app.route('/rotateNeg')
```

Comment:
Fore the future, if we intend to have many routes, it may be better to have the routes like `/science/numSections`

Reply:
fair

Reply:
You didn't update it though `numSection` remains